recipes/pycalphad/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "0.4.1" %}
+{% set sha256 = "2603069927f4e6f69038f90a993f810c63585b6bf428473bcd5254d940a449f" %}
```

Comment:
Yeah, I'd try to have another check other than the CIs for good measure. Sometimes they have download issues. Worst case something nefarious has happened. Though I don't have much reason to suspect it would. It is a bit unfortunate that we don't have someone else providing us the verified checksum.